Adyen.Test/BaseTest.cs

```diff
@@ -341,13 +355,15 @@ protected Client CreateMockTestClientApiKeyBasedRequest(string fileName)
         /// <returns>IClient implementation</returns>
         protected Client CreateMockTestClientPosCloudApiRequest(string fileName)

```

Comment:
This method is used by the point of sale (POS). In your changes you added the async methods for ecomm functionality.